src/xcode/ENA/ENAUITests/HealthCertificate/ENAUITests_13_CreateHealthCertificate.swift

```diff
@@ -102,38 +109,45 @@ class ENAUITests_13_CreateHealthCertificate: CWATestCase {
 
 		let certificateTitle = try XCTUnwrap(app.cells[AccessibilityIdentifiers.HealthCertificate.Overview.vaccinationCertificateCell])
 		
-		snapshot("screenshot_certificate_overview_vaccination_blue")
+		snapshot("screenshot_vaccination_certificate_overview")
 		certificateTitle.waitAndTap()
 
 		let qrCodeCell = app.cells[AccessibilityIdentifiers.HealthCertificate.qrCodeCell]
 		XCTAssertTrue(qrCodeCell.waitForExistence(timeout: .short))
 
-		snapshot("screenshot_health_certificate_blue")
-
+		snapshot("screenshot_vaccination_certificate_overview_part1")
+		app.swipeUp(velocity: .slow)
+		snapshot("screenshot_vaccination_certificate_overview_part2")
+		
 		// Certified Person screen
 		let certificateCells = try XCTUnwrap(app.cells[AccessibilityIdentifiers.HealthCertificate.Person.certificateCell])
 		XCTAssertTrue(certificateCells.waitForExistence(timeout: .short))
 		XCTAssertEqual(app.cells.matching(identifier: AccessibilityIdentifiers.HealthCertificate.Person.certificateCell).count, 2)
 	}
 
-	/*
 	func test_screenshot_TestCertificate() throws {
 		app.setLaunchArgument(LaunchArguments.infoScreen.healthCertificateInfoScreenShown, to: true)
 		app.setLaunchArgument(LaunchArguments.healthCertificate.testCertificateRegistered, to: true)
 		app.launch()
 
 		// Navigate to Certificates Tab.
 		app.buttons[AccessibilityIdentifiers.TabBar.certificates].waitAndTap()
+		
+		// Navigate to the test certificate overview screen
+		app.cells[AccessibilityIdentifiers.HealthCertificate.Overview.vaccinationCertificateCell].waitAndTap()
```

Comment:
I would maybe call it `Navigate to the person screen` as it is not only test certificates and the screen that you are on is already called `Overview` (as you can see on the accessibility identifier that contains `.Overview.`)

And it would be cool if you could rename the identifier here from `vaccinationCertificateCell` to `healthCertifiedPersonCell` as it is not only representing vaccinations by now

Reply:
Will do it.